src/core/Quote.ml

```diff
@@ -1,4 +1,4 @@
-module S = Syntax
+module S = Syntax 
```

Comment:
Mortal spacing detected. How about...
```vim
'ocaml': ['ocp-indent', 'remove_trailing_lines', 'trim_whitespace']
```

Reply:
Ahh, I am using Vim but when my computer died I lost my configurations... @cangiuli Can you tell me the vim-equivalent to do the following on save:

1. Run whatever the registered formatter is (this works for me when I highlight text and press `=`, but I want it run when I save)
2. Trim the whitespace, etc.

Reply:
This would work for OCaml files and also trim the whitespace for other files:
```vim
let g:ale_fix_on_save = 1
let g:ale_fixers = {
      \ 'ocaml': ['ocp-indent', 'remove_trailing_lines', 'trim_whitespace'],
      \ '*': ['remove_trailing_lines', 'trim_whitespace']
      \ }
```

Reply:
Wonderful, thank you!

Reply:
PS: this would destroy the `.mly` file. I always had to `:let g:ale_fix_on_save=0` when unleashing the genie...

Reply:
LOL is there a way to disable this for `.mly`? LOL

Reply:
Maybe. I am too lazy. LOL

Reply:
You should be grateful that the genie is NOT coming out too often.

Reply:
LOL